poetry/templates/404.html

```diff
@@ -0,0 +1,29 @@
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"32px","bottom":"32px"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group" style="padding-top:32px;padding-bottom:32px">
+    <!-- wp:group {"style":{"spacing":{"blockGap":"64px"}},"layout":{"type":"flex","flexWrap":"nowrap"}} -->
+    <div class="wp-block-group">
+        <!-- wp:group {"style":{"spacing":{"blockGap":"88px"}},"layout":{"type":"flex","orientation":"vertical"}} -->
+        <div class="wp-block-group">
+            <!-- wp:site-title {"level":0,"isLink":false,"style":{"typography":{"fontSize":"24px","fontStyle":"normal","fontWeight":"600"}},"textColor":"custom-text"} /-->
```

Comment:
I think it would be better to keep the header element like the index.html template has it. It's more consistent between pages and also we need to keep the `header` element for accessibility reasons. This comment goes for both templates

Reply:
Ok, I forgot the header in poetry was super huge. My recommendation still remains, let's make a secondary header pattern or template part to reuse in internal pages that doesn't have the huge text, but let's keep it consistent between pages. And let's use the tagName header on it so we don't miss the header element